src/main/kotlin/frc/robot/subsystems/swerve/SwerveDrive.kt

```diff
@@ -178,6 +178,28 @@ class SwerveDrive private constructor
         estimator.resetPosition(pose.rotation, modulePositions, pose)
     }
 
+    private fun isSkidding(): Boolean {
+        var weightedAvg = 0.0
+        var denominator = 0.0
+        val slipRatios = Array<Double>(modules.size)
+        {
+            ((modules[it]?.velocity ?: 0.0) - chassisSpeeds.getVelocityMagnitude())
+                .div(chassisSpeeds.getVelocityMagnitude())
```

Comment:
Please make a few variables here, it's very hard to read. Also, why `div()`? Why not simply `/`?

Reply:
c763355bfc04aefc5b9b1086d1d3842ddced9612

Reply:
You don't need to include "magnitude", because 
> scalar absolute value (magnitude) of velocity is called speed, [[Wikipedia](https://en.wikipedia.org/wiki/Velocity#:~:text=Velocity%20is%20a%20physical%20vector,m%E2%8B%85s%E2%88%921).)]

Reply:
456ed6c3d9287839f805f268c0966528c390f9da

Reply:
456ed6c3d9287839f805f268c0966528c390f9da